fedimg/services/ec2/ec2imguploader.py

```diff
@@ -255,12 +258,15 @@ def clean_up(self, image_id, delete_snapshot=True, force=False):
         if self.volume:
             self._connect().destroy_volume(self.volume)
 
-        self._connect().deregister_image(
-            image_id,
-            delete_snapshot=delete_snapshot
-        )
+        # self._connect().deregister_image(
```

Comment:
Please delete this code rather than leaving it commented out. We can always find it in git's history.

Reply:
Yes! I added the WIP tag to this PR. I need to do a few more changes to this PR